src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -66,4 +66,15 @@ public static int countNegative(int[] array) {
                 .filter(value -> value < 0)
                 .count();
     }
+
+    public static void reverse(int[] array) {
+        checkNotNull(array);
```

Comment:
checkNotNull accepts the second parameter - a message. Please, add a descriptive message (i.e. "Array must not be null")